sigle/src/modules/editor/TipTapEditor.tsx

```diff
@@ -45,6 +45,10 @@ const fadeInAnimation = keyframes({
 });
 
 const StyledEditorContent = styled(EditorContent, {
+  '& code': {
```

Comment:
as this is related to the rendering of the blocks would be better to move to the tailwind package right?